internal/property/property.go

```diff
@@ -30,6 +30,10 @@ func (p Property) Validate() error {
 	return nil
 }
 
+func (p Property) String() string {
+	return fmt.Sprintf("type: %s, value: %s", p.Type, p.Value)
```

Comment:
In case either is an empty string
```suggestion
	return fmt.Sprintf("type: %q, value: %q", p.Type, p.Value)
```